front/src/applications/editor/tools/switchEdition/tool.tsx

```diff
@@ -49,7 +50,15 @@ const SwitchEditionTool: Tool<SwitchEditionState> = {
         icon: AiFillSave,
         labelTranslationKey: 'Editor.tools.switch-edition.actions.save-switch',
         isDisabled({ isLoading, state }) {
-          return state.portEditionState.type !== 'idle' || isLoading || false;
+          const ports = filter(state.entity?.properties?.ports ?? {}, (p) => !!p?.track);
+          const detectedDuplicates = filter(groupBy(ports, 'track'), (v, _) => v.length > 1);
+          return (
+            state.portEditionState.type !== 'idle' ||
+            ports.length === 0 ||
+            !!detectedDuplicates.length ||
+            isLoading ||
+            false
+          );
```

Comment:
With these checks, the save button can be pressed as long as one port is selected. Even though there isn't any endpoint call and the missing fields are disabled when submitting, I think we could rather keep the button disabled as long as all the ports aren't selected.

```suggestion
          const ports = filter(state.entity?.properties?.ports ?? {}, (p) => !!p?.track);
          const isMissingPort =
            state.entity.properties &&
            Object.values(state.entity.properties.ports).some((port) => isEmpty(port));
          const detectedDuplicates = filter(groupBy(ports, 'track'), (v, _) => v.length > 1);
          return (
            state.portEditionState.type !== 'idle' ||
            isMissingPort ||
            !!detectedDuplicates.length ||
            isLoading ||
            false
          );
```